### PR TITLE
add validation information for server tags

### PR DESCRIPTION
**Pull request by yirensum**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-740.surge.sh

File: modules/ROOT/pages/clustering/servers.adoc
Comment (renetapopova) [inline modules/ROOT/pages/clustering/servers.adoc:156]:
```suggestion
Therefore, for server tags that need a larger character set (e.g. Chinese or Arabic), it is recommended to use Cypher to alter server tags.
====

```

File: modules/ROOT/pages/clustering/servers.adoc
Comment (renetapopova) [inline modules/ROOT/pages/clustering/servers.adoc:154]:
```suggestion
[NOTE]
====
The input for server tags is a comma-separated list which cannot have duplicates.
```

File: modules/ROOT/pages/clustering/servers.adoc
Comment (renetapopova) [inline modules/ROOT/pages/clustering/servers.adoc:155]:
```suggestion
Also, the encoding of _.conf_ files is *Latin1*.
```

**Review by renetapopova — APPROVED** [review]
Thanks, @yirensum! Some small suggestions, otherwise looks good.

**Comment by renetapopova** [comment]
@yirensum, which version should this PR be merged into?

**Comment by yirensum** [comment]
> @yirensum, which version should this PR be merged into?

Thanks for the review @renetapopova, so this will go in as part of the 5.8.0 release (which I believe is cutoff end of this week?)

Also, I'll make one change - apparently Kernel can now read .conf files using encoders other than Latin-1. Will push a commit in a bit.

**Comment by yirensum** [comment]
Thanks again for the review @renetapopova, I think this is good to go from my side. Is this only allowed to be merged after cut-off hence the `Do not merge` label?

**Pull request merged**
